Source/WebKitLegacy/mac/WebCoreSupport/WebFrameLoaderClient.mm

```diff
@@ -1553,10 +1553,11 @@ static void addRedirectURL(WebHistoryItem *item, const String& url)
         auto element = adoptNS([[WebElementDictionary alloc] initWithHitTestResult:core(m_webFrame.get())->eventHandler().hitTestResultAtPoint(mouseEventData->absoluteLocation, hitType)]);
         [result setObject:element.get() forKey:WebActionElementKey];
 
-        if (mouseEventData->isTrusted)
-            [result setObject:@(mouseEventData->button) forKey:WebActionButtonKey];
-        else
-            [result setObject:@(WebCore::NoButton) forKey:WebActionButtonKey];
+        const auto toUnderlyingType = [](MouseButton button) {
+            return static_cast<std::underlying_type_t<MouseButton>>(button);
+        };
+        auto button = mouseEventData->isTrusted ? toUnderlyingType(mouseEventData->button) : toUnderlyingType(MouseButton::NoButton);
```

Comment:
we already have `WTF::enumToUnderlyingType` that you can use I think

Reply:
Oh, TIL. I'll change it